cloudbuild.yaml

```diff
@@ -16,35 +16,34 @@ steps:
   - name: gcr.io/cloud-builders/git
     args: [ submodule, update, --init, --recursive ]
     id: fetch-git-submodules
+    waitFor:
+      - "-"
```

Comment:
Maybe add a comment here explaining this. Maybe this is obvious to people who use cloudbuild, but it's not obvious to me compared to other usages of waitFor.

Reply:
Excellent suggestion - will do.